# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/other/PDocAccessible.ipdl
# Commit: 4572c1ba670a
# Full Hash: 4572c1ba670a3ddb159cf37ba9684fb6055f260e
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2021-01-30 05:43:48
# Regressor Bug: 1688972
# File Overlap Count: 1
# Description:
#   Bug 1688972 - Make FocusedChild work across document/process boundaries. r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D103102
# ==============================================================================

diff -r f262a5526f6c -r 4572c1ba670a accessible/ipc/other/PDocAccessible.ipdl
--- a/accessible/ipc/other/PDocAccessible.ipdl	Fri Jan 29 22:44:30 2021 +0200
+++ b/accessible/ipc/other/PDocAccessible.ipdl	Fri Jan 29 20:57:30 2021 +0000
@@ -310,7 +310,7 @@
 
   async TakeFocus(uint64_t aID);
   nested(inside_sync) sync FocusedChild(uint64_t aID)
-    returns(uint64_t aChild, bool aOk);
+    returns(nullable PDocAccessible aResultDoc, uint64_t aResultID);
 
   nested(inside_sync) sync Language(uint64_t aID) returns(nsString aLocale);
   nested(inside_sync) sync DocType(uint64_t aID) returns(nsString aType);